src/Test.UnitTests.Sarif.Driver/Sdk/AnalyzeCommandBaseTests.cs

```diff
@@ -1191,6 +1259,7 @@ public void MultithreadedAnalyzeCommandBase_TargetFileSizeTestCases()
                 mockFileSystem.Setup(x => x.FileOpenRead(It.IsAny<string>())).Returns(mockStream.Object);
                 mockFileSystem.Setup(x => x.FileExists(tempFile.Name)).Returns(true);
                 mockFileSystem.Setup(x => x.FileInfoLength(It.IsAny<string>())).Returns((long)testCase.fileSizeInBytes);
+                mockFileSystem.Setup(x => x.IsSymbolicLink(It.IsAny<string>())).Returns((bool)testCase.isSymbolicLink);
```

Comment:
The above is a very good test, but it's mocking out the symbolic link detection and there isn't any coverage making sure that is done correctly.  This change would really benefit from an augmentation to an E2E test that incorporates symbolic links, especially in a case where the only targets are symlinks.  If that's not practicable, then direct UT coverage of FileSystem.IsSymbolicLink would be helpful.

Reply:
Added a test for FileSystem.IsSymbolicLink.